src/plugin.ts

```diff
@@ -9,33 +9,35 @@ import { adjustCanvas, parseImage } from './utils/image'
 import { logger } from './utils/logger'
 
 export type DiffOption = 'always' | 'fail' | 'never'
+export type TypeOption = 'regression' | 'base'
 
 export type VisualRegressionOptions = {
-  type: string
+  /** kind of comparison that we are going to execute */
+  type: TypeOption
```

Comment:
With this comment on top of a key you can add annotations to the type , very cool
![image](https://github.com/cypress-visual-regression/cypress-visual-regression/assets/2030605/c4e66a1e-843c-4eb8-9ba9-4fcb61f9496d)